Extension/src/LanguageServer/ui.ts

```diff
@@ -426,12 +433,16 @@ export class LanguageStatusUI {
     private showConfigureIntelliSenseButton: boolean = false;
     private configureIntelliSenseTimeout?: NodeJS.Timeout;
 
-    public async ShowConfigureIntelliSenseButton(show: boolean, client?: Client): Promise<void> {
+    public async ShowConfigureIntelliSenseButton(show: boolean, client?: Client, configurationType?: ConfigurationType, sender?: string): Promise<void> {
+        if (configurationType !== undefined && sender !== undefined) {
+            const showButton: string = show ? 'true' : 'false';
+            telemetry.logLanguageServerEvent('showConfigureIntelliSenseButton', { configurationType, sender, showButton });
```

Comment:
I see this telemetry getting sent 8 times or so for a 2 workspace folder project. I think maybe the check `client !== this.currentClient` that is done below should be added beforehand.

Reply:
That would be fine, but we do want to log telemetry before exiting the function, so the check for the experiment to exit early should remain below.

Reply:
This issue wasn't resolved by the changes.

Reply:
I fixed it.